ext/mixed/js/core.js

```diff
@@ -425,11 +423,8 @@ const yomichan = (() => {
             };
         }
 
-        _getProblemMessage(type) {
-            switch (type) {
-                case 'unhandled-rejection': return 'has encountered an unhandled promise rejection.';
-                default: return 'has encountered a problem.';
-            }
+        _getProblemMessage() {
```

Comment:
`_getProblemMessage` is still called with `type` at line 399

Reply:
Fixed 8a203947e2f3a89f761c1f62896236accd5cd0a1